ClosedXML.Tests/Excel/ImageHandling/PictureTests.cs

```diff
@@ -470,5 +471,26 @@ public void CanCopyEmfPicture()
             img2 = ws2.Pictures.First();
             Assert.AreEqual(XLPictureFormat.Emf, img2.Format);
         }
+
+        [Test]
+        public void CanKeepPictureShapeOrder()
+        {
+            using var stream = TestHelper.GetStreamFromResource(TestHelper.GetResourcePath(@"Examples\ImageHandling\ImageShapeOrder.xlsx"));
+            using var wb = new XLWorkbook(stream);
+            var ws1 = wb.Worksheets.First();
+            var img1 = ws1.Pictures.First();
+            var xlImage1 = img1 as ClosedXML.Excel.Drawings.XLPicture;
+            var relId = xlImage1.RelId;
+            int pos1 = OpenXmlHelper.FindImageRelIdDrawingOrder(stream,ws1.Name, relId);
+
+            using var ms = new MemoryStream();
+            wb.SaveAs(ms);
+            stream.Close();
+            int pos2 = OpenXmlHelper.FindImageRelIdDrawingOrder(ms, ws1.Name, relId);
+            ms.Close();
+
+            Assert.AreEqual(pos1, pos2);
```

Comment:
Since there isn't an API for Z-order, it is better to just compare files. In this case, we are basically checking cycle load-save and file comparison is a standard solution for that. Test shouldn't deal with xlsx xml structure directly.